### DISCUSSION TITLE
Tool for Theme Preview

### DISCUSSION THREAD

**Discussion by MajorTomDE** [ventoy/Ventoy#2171] [Ideas]
Editing theme.txt and so on is relatively simple, but it's so annoying to always restart the notebook to get an theme preview.

My suggestion is a windows tool to get an theme preview without notebook restarting.

**Comment by steve6375** [comment]
Most people use a vm like virtual box and simply reset the VM.  Very quick to test.

**Comment by MajorTomDE** [comment]
@steve6375
You made my day. I found a great tutorial and it works fine: [Virtual Box - Boot from USB Stick](https://www.youtube.com/watch?v=zL9ib7tvew8)

Thank you!!!

**Reply by steve6375** [reply]
That is ok for ventoy probably, but there are some issues with writes to the usb drive under low level boot environments like dos, grub, grub4dos, etc. Vbox snapshots the usb drive because the drive is not released by windows.
Look for my YouTube video on vbox and vmub which was specifically developed to work around those issues. Vmub is only compatible with vbox 5 though but it's really easy to use and you can have many usb drives.
Vmub also has a trick. If you hold down the ctrl key and then click start  does not lock the drive, this is ideal for testing themes, etc because you can edit the file on the drive and simply reboot the VM.
https://rmprepusb.com/tutorials/004-how-to-boot-directly-from-a-usb-drive-using-an-emulator-or-vm-under-windows/

**Comment by rahaaatul** [comment]
Do you have a issue with small resolution? I can't seem to make the screen bigger without manually going to `F5: Tools` > `Resolution Configuration` and change the resolution every time I boot.
